local-volume/bootstrapper/README.md

```diff
@@ -117,11 +117,20 @@ make push
 Deploy to existing cluster:
 
 ```console
-kubectl create -f deployment/kubernetes/example-config.yaml
-kubectl create -f deployment/kubernetes/admin-account.yaml
-kubectl create -f deployment/kubernetes/bootstrapper.yaml
+kubectl create -f deployment/kubernetes/<version>/example-config.yaml
+kubectl create -f deployment/kubernetes/<version>/admin-account.yaml
+kubectl create -f deployment/kubernetes/<version>/bootstrapper.yaml
 ```
 
+The version in the path above can be `latest` for the latest provsioner or one of the earlier images.
```

Comment:
Can you also update this readme, and the main readme noting that the configmap is required.

Reply:
@msau42 Fixed.

Reply:
There's a section at the top that talks about how we create a default configmap. Can you remove it

Reply:
fixed.